data/langdb.yaml

```diff
@@ -488,6 +488,7 @@ languages:
   twd: [Latn, [EU], Tweants]
   ty: [Latn, [PA], reo tahiti]
   tyv: [Cyrl, [AS], тыва дыл]
+  tzl: [Latn, [WW], Talossan]
```

Comment:
Do you want to map this is a word wide spoken language? I don't see any supporting reference for that.

https://en.wikipedia.org/wiki/Talossa indicate 'AM' as the correct region. Am I right?